Documentation/supermuc.rst

```diff
@@ -0,0 +1,362 @@
+
+::
+
+  Host supermuNG
+     Hostname sks.supermuc.lrz.de
+     User <Your Login>    
+     RemoteForward ddddd github.com:22
+
+where ddddd is an arbitrary 5-digital port number.
+
+2. ssh supermuNG to login to supermuc. The add the following lines to the ~/.ssh/config:
+
+:: 
+
+  Host github
+     HostName localhost
+     User git
+     Port ddddd
+    
+With ddddd the same port number as before.
+
+4. Create SSH key by typing 
+
+::
+
+   ssh-keygen -t rsa 
+
+5. Go to https://github.com/settings/ssh, add a new SSH key, pasting the public key you just created on supermuc  ~/.ssh/id_rsa.pub. 
+Logout of supermuc and log back in (ssh supermuc2). you should now be able to clone SeisSol using:
+
+
+::
+
+  git clone git@github:SeisSol/SeisSol.git  
+
+
+Pay attention to the git clone address ('https://github.com/' replaced by 'git@github:'). 
+If it works, you will see several lines of ‘cloning ….’.
+
+Now for initializing the submodule folder, copy the following to fix_submodules.sh and run it within SeisSol.
+
+::
+
+  #!/bin/bash                                                                                                            
+  git submodule init
+  for module_full in `git config -l | awk -F"=" {'print $1'} | grep submodule ` ;
+  do
+   echo $module_full
+   module=$(git config ${module_full} | sed "s/.*github\.com\///")
+   if [ "$module" != "true" ]; then
+      echo $module
+      echo "git config ${module_full} git@github:${module}"
+      git config ${module_full} git@github:${module}
+   fi
+  done
+  git submodule update
+ 
+
+Supermuc-NG
+===========
+
+1. git clone git@github:SeisSol/SeisSol.git  
+
+2. Load module. Could add these lines to .bashrc:
+
+::
+
+  ##### module load for SeisSol
+  module load scons cmake/3.6 python/2.7_intel slurm_setup
+  module load parmetis/4.0.3-intel-impi-i64-r64 metis/5.1.0-intel-i64-r64
+  module load netcdf/4.6.1-intel-impi-hdf5v1.8-parallel hdf5/1.8.20-intel-impi
+  module switch spack/staging/19.1 spack/master
+  module switch devEnv/Intel devEnv/Intel/2019
+  module load libszip/2.1.1
+
+  ####### universal setup for SeisSol
+  export PATH=~/local/bin:$PATH
+  export PKG_CONFIG_PATH=~/local/lib/pkgconfig/:$PKG_CONFIG_PATH
+  export LD_LIBRARY_PATH=~/local/lib:$PARMETIS_LIB:$METIS_LIB:$NETCDF_BASE/lib:$HDF5_BASE/lib:$LD_LIBRARY_PATH
+  export CPATH=~/local/include:$PARMETIS_INC:$METIS_INC:$NETCDF_BASE/include:$HDF5_BASE/include:$CPATH
+  export LIBRARY_PATH=~/local/lib:$PARMETIS_LIB:$METIS_LIB:$NETCDF_BASE/lib:$HDF5_BASE/lib:$LIBRARY_PATH
+    
+  ####  local setup for SeisSol. I installed libxsmm and ASAGI in my own folder (/dss/dsshome1/02/di52lak2/myLib). 
+  export PATH=/dss/dsshome1/02/di52lak2/myLib/libxsmm/bin:$PATH
+  export PKG_CONFIG_PATH=/dss/dsshome1/02/di52lak2/myLib/ASAGI/build/lib/pkgconfig:$PKG_CONFIG_PATH
+  export LD_LIBRARY_PATH=/dss/dsshome1/02/di52lak2/myLib/ASAGI/build/lib:$LD_LIBRARY_PATH
+
+
+3. Install libxsmm and ASAGI (Optional)
```

Comment:
This line could be confusing, as it could be interpreted as implying that both libxsmm and ASAGI are optional